UnitedSets/Controls/CellVisualizer.xaml

```diff
@@ -78,7 +78,7 @@
                 </Rectangle>
             </Grid>
         </DataTemplate>
-        <DataTemplate x:Name="VerticalCellDataTemplate" x:DataType="class:Cell">
```

Comment:
Okay, I guess that work but I would rename them to `HorizontalSplitCellDataTemplate` and `VerticalSplitCellDataTemplate` after merging to avoid confusion

Reply:
See my comment in the PR itself, as this is still not quite working properly and I don't know why.